deploy/warehouse/target/data.sql

```diff
@@ -5,7 +5,26 @@ begin;
 
 insert into warehouse.organism (lineage, identifiers, details)
     values
+        ('Adenovirus',              'NCBITAXON => 10508',       null),
         ('Bordetella_pertussis',    'NCBITAXON => 520',         '{"report_to_public_health": true}'),
+        ('Chlamydophila_pneumoniae','NCBITAXON => 83558',       null),
+        ('Enterovirus',             'NCBITAXON => 12059',       null),
+        ('Enterovirus.D',           'NCBITAXON => 138951',      null),
+        ('Enterovirus.D.68',        'NCBITAXON => 42789',       null),
+        ('Enterovirus.Rhinovirus',  null,                       null),
```

Comment:
This strikes me as a little weird, but indeed it's how NCBI Taxonomy structures it and how Wikipedia notes it is defined cladistically. I think the question is if we expect to include rhinoviruses when querying for all `Enterovirus`. @trvrb may have thoughts/opinions here.

Reply:
Agreed. Jover and I discussed this and were unsure whether we are more concerned with representing true taxonomies or with reporting organisms colloquially.

Reply:
I've pinged Slack about this: https://seattle-flu-study.slack.com/archives/CCAA9RBFS/p1570550480065400

Reply:
Let's call this just `Rhinovirus` for now.

Reply:
It is now just `Rhinovirus`.